plugins/base/src/main/resources/dokka/styles/style.css

```diff
@@ -192,8 +192,6 @@ html ::-webkit-scrollbar-thumb {
 }
 
 .cover > .platform-hinted {
-    padding-top: 12px;
-    margin-top: 12px;
```

Comment:
This is not even noticable since there's bottom padding from `h1.cover` already. Makes the page a little bit more compact and grabs less attention